### ISSUE TITLE
Rope becomes very long and broken

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmith/hedgewars#482]
```
What steps will reproduce the problem?
Here's the demo which can reproduce it: http://unit22.org/hh/rope_bug.43.hwd
Rewind it to 10:30.
I don't know how to reproduce it manually.

What is the expected output? What do you see instead?
Normal rope behaviour, i see http://unit22.org/hh/rope_bug.png instead. If 
player would release rope he would be stuck in the middle of the map.

What version of the product are you using? On what operating system?
0.9.18, arch linux 64bit

Please provide any additional information below.

It's old bug and it happens very rarely.
```

Original issue reported on code.google.com by `ben...@unit22.org` on 29 Nov 2012 at 9:06

**Comment by GoogleCodeExporter** [comment]
```
Thank you for your bug report bender!

It's awesome that you provided a demo, but please don't forget to also link the 
custom map it is played on, thanks :P
```

Original comment by `sheepyluva` on 15 Jan 2013 at 5:31

**Comment by GoogleCodeExporter** [comment]
```
Oh right. Sometimes i forget that not everyone has my map pack :) It's ShBall, 
here's the link http://hh.unit22.org/addon/123. It's in unit22 repo in DLC too.
```

Original comment by `ben...@unit22.org` on 15 Jan 2013 at 6:08

**Comment by GoogleCodeExporter** [comment]
```
Thank you!

I attached a picture of what happens right before the bug. Maybe it can help 
figuring out how to reproduce this.

Note 1: the last two frames are both very very briefly before the rope_bug.png 
and they are hardly visible to the human eye. I hope I inserted them in the 
right order.

Note 2: the bug occurs regardless of if the weapon crate on the top exists or 
not
(tested by making the crate vanish by inserting
if (GameTicks = 632574) then Gear^.Damage := 1;
at the begin of the doStepCase procedure)
```

Original comment by `sheepyluva` on 16 Jan 2013 at 10:23
* Changed state: **Accepted**
* Added labels: **Milestone-Release1.0**

Attachments:
* [whoops.png](https://storage.googleapis.com/google-code-attachments/hedgewars/issue-482/comment-3/whoops.png)
* [rope_bug.png](https://storage.googleapis.com/google-code-attachments/hedgewars/issue-482/comment-3/rope_bug.png)

**Comment by GoogleCodeExporter** [comment]
```
Note 3: the rope in the last frame of whoops.png might be longer than displayed 
(maybe as long as in rope_bug.png), I just noticed the screenshot was cut. I'll 
see if I can take a better one.
```

Original comment by `sheepyluva` on 16 Jan 2013 at 10:30

**Comment by GoogleCodeExporter** [comment]
Original comment by `sheepyluva` on 17 Jan 2013 at 1:08
* Added labels: **ReleaseBug-0.9.18**

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision 5b273af3ac95.
```

Original comment by `unC0Rr` on 17 Jan 2013 at 8:43
* Changed state: **Fixed**
